stdlib/Logging/src/ConsoleLogger.jl

```diff
@@ -0,0 +1,237 @@
+"""
+    ConsoleLogger(stream::IO; min_level=Info, catch_exceptions=true)
+
+A logger designed for readable log formatting at the julia REPL. `min_level` is
+the default log level for all modules.
+
+If `catch_exceptions` is set to `true`, the logger will log any exceptions
+generated during log message processing at `Error` level.  Setting this to
+`false` can be a helpful way to debug errors in your log statements.
+"""
+mutable struct ConsoleLogger <: AbstractLogger
+    stream::IO
+    default_min_level::LogLevel
+    catch_exceptions::Bool
+    prev_progress_key
+    message_counts::Dict{Any,Int}
+    module_limits::Dict{Module,LogLevel}
+    blacklisted_ids::Set{Any}
+end
+
+function ConsoleLogger(stream::IO; min_level=Info, catch_exceptions=true)
+    ConsoleLogger(stream, parse_level(min_level), catch_exceptions, nothing,
+             Dict{Any,Int}(), Dict{Module,LogLevel}(), Set{Any}())
+end
+
+"""
+    configure_logging(logger::ConsoleLogger, [_module]; min_level=Info)
```

Comment:
GitHub code formatting flips out from the `_module` variable name, same thing happens here: https://github.com/JuliaLang/julia/blob/fbbdaeb0465c29c5cb06b7c0e778e74ef84ceb1d/base/logging.jl#L30 and later in this file.

Reply:
Oh, so *that's* what's going on.  Looks like it's already fixed:

https://github.com/JuliaEditorSupport/atom-language-julia/issues/134